src/tomli/_parser.py

```diff
@@ -118,7 +178,7 @@ def loads(__s: str, *, parse_float: ParseFloat = float) -> dict[str, Any]:  # no
                 pos, header = create_dict_rule(src, pos, out)
             pos = skip_chars(src, pos, TOML_WS)
         elif char != "#":
-            raise suffixed_err(src, pos, "Invalid statement")
+            raise TOMLDecodeError(msg="Invalid statement", doc=src, pos=pos)
```

Comment:
Can't pass them positionally now?

Reply:
Oh forgot about that, yes we can now. I'll edit these, thanks!